packages/twenty-front/src/modules/prefetch/components/PrefetchDataProvider.tsx

```diff
@@ -2,11 +2,16 @@ import React from 'react';
 
 import { PrefetchRunQueriesEffect } from '@/prefetch/components/PrefetchRunQueriesEffect';
 
+export const PrefetchContext = React.createContext({
```

Comment:
let's avoid this pattern, it will trigger re-renders. Instead, we should store this prefetch loading state in a recoil state and fetch it as low as possible.